inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -569,6 +569,10 @@ std::shared_ptr<ngraph::Node> V10Parser::createNode(const std::vector<ngraph::Ou
         if (pr_data) {
             rtInfo["PrimitivesPriority"] = std::make_shared<::ngraph::VariantWrapper<std::string> >(pr_data.value());
         }
+        const auto cr_data = dn.attribute("ConvReshape");
```

Comment:
Why do we need to support this attribute in the parser?

Who should define this runtime information? If user should do it, should we describe this runtime attribute to the documentation?

Reply:
@taka-no-me , could you please comment it?